clouddq/classes/dq_entity_uri.py

```diff
@@ -179,3 +179,10 @@ def _validate_bigquery_uri_fields(self, config_id: str, configs: dict) -> None:
                     f"input config under headings 'metadata_default_registries', e.g.\n"
                     f"{SAMPLE_DEFAULT_REGISTRIES_YAML}",
                 )
+
+    def get_table_name_from_entity_uri(self: EntityUri):
```

Comment:
maybe just `get_table_name` is sufficient? the entity_uri part if self-explanatory from the parent class.

Reply:
have you changed this method name as well @AmandeepSinghCS ?

Reply:
@thinhha yes this is renamed